src/views/ModelPlan/CollaborationArea/Cards/ModelPlanCard/index.tsx

```diff
@@ -0,0 +1,161 @@
+import React, { useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import {
+  Button,
+  Card,
+  CardBody,
+  CardFooter,
+  CardHeader
+} from '@trussworks/react-uswds';
+import {
+  GetModelPlanQuery,
+  TaskStatus,
+  useGetModelPlanQuery
+} from 'gql/gen/graphql';
+
+import UswdsReactLink from 'components/LinkWrapper';
+import Modal from 'components/Modal';
+import { Avatar } from 'components/shared/Avatar';
+import ShareExportModal from 'components/ShareExport';
+import { formatDateLocal } from 'utils/date';
+import {
+  getLatestModifiedDate,
+  ITSolutionsType,
+  StatusMessageType
+} from 'views/ModelPlan/TaskList';
+import { TaskListStatusTag } from 'views/ModelPlan/TaskList/_components/TaskListItem';
+
+import './index.scss';
+
+type GetModelPlanTypes = GetModelPlanQuery['modelPlan'];
+type OperationalNeedsType = GetModelPlanQuery['modelPlan']['operationalNeeds'][0];
+
+type ModelPlanCardType = {
+  modelID: string;
+  setStatusMessage: (message: StatusMessageType) => void;
+};
+
+const ModelPlanCard = ({ modelID, setStatusMessage }: ModelPlanCardType) => {
+  const { t: modelPlanCardT } = useTranslation('modelPlanCard');
+  const [isExportModalOpen, setIsExportModalOpen] = useState<boolean>(false);
+  const { data, loading } = useGetModelPlanQuery({
+    variables: {
+      id: modelID
+    }
+  });
+
+  const modelPlan = data?.modelPlan || ({} as GetModelPlanTypes);
+  const {
+    modifiedDts,
+    modifiedByUserAccount,
+    basics,
+    generalCharacteristics,
+    participantsAndProviders,
+    beneficiaries,
+    opsEvalAndLearning,
+    payments,
+    operationalNeeds = [],
+    taskListStatus
+  } = modelPlan;
+
+  const getITSolutionsStatus = (
+    operationalNeedsArray: OperationalNeedsType[]
+  ) => {
+    const inProgress = operationalNeedsArray.find(need => need.modifiedDts);
+    return inProgress ? TaskStatus.IN_PROGRESS : TaskStatus.READY;
+  };
+
+  const itSolutions: ITSolutionsType = {
+    modifiedDts: getLatestModifiedDate(operationalNeeds),
+    status: getITSolutionsStatus(operationalNeeds)
+  };
+
+  // Returns the number of sections that have been started (i.e. not in 'READY' status)
+  const sectionStartedCounter = () => {
+    if (loading) return 0;
+
+    const sections = [
+      basics.status,
+      generalCharacteristics.status,
+      participantsAndProviders.status,
+      beneficiaries.status,
+      opsEvalAndLearning.status,
+      payments.status,
+      itSolutions.status
+    ];
+
+    return sections.filter(status => status !== TaskStatus.READY).length;
+  };
```

Comment:
This function will get created on every render, which isn't not right, but also not sure if it needs to be a function

``
  const count = useMemo(() => {
    // Returns the number of sections that have been started (i.e. not in 'READY' status)
    const sections = [
      basics.status,
      generalCharacteristics.status,
      participantsAndProviders.status,
      beneficiaries.status,
      opsEvalAndLearning.status,
      payments.status,
      itSolutions.status
    ];

    return sections.filter(status => status !== TaskStatus.READY).length || 0;
  }, [
    basics.status,
    generalCharacteristics.status,
    participantsAndProviders.status,
    beneficiaries.status,
    opsEvalAndLearning.status,
    payments.status,
    itSolutions.status
  ]);```